macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -113,6 +113,7 @@
                 column_name,
                 metric_name,
                 case
+                    when training_stddev is null then null
```

Comment:
Make sure this works for all platforms, we might need to pass null casted as int

Reply:
Working on all platforms 🙂